modules/images-update-global-pull-secret.adoc

```diff
@@ -14,9 +14,9 @@ You can update the global pull secret for your cluster by either replacing the c
 
 [IMPORTANT]
 ====
-To transfer your cluster to another owner, you must first initiate the transfer in link:https://console.redhat.com/openshift/[Red Hat OpenShift Cluster Manager], and then update the pull secret on the cluster. Updating a cluster's pull secret without initiating the transfer in OpenShift Cluster Manager causes the cluster to stop reporting Telemetry metrics in OpenShift Cluster Manager. 
+To transfer your cluster to another owner, you must first initiate the transfer in {cluster-manager-url}, and then update the pull secret on the cluster. Updating a cluster's pull secret without initiating the transfer in {cluster-manager} causes the cluster to stop reporting Telemetry metrics in {cluster-manager}.
 
-For more information link:https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html/managing_clusters/assembly-managing-clusters#transferring-cluster-ownership_assembly-managing-clusters[about transferring cluster ownership], see "Transferring cluster ownership" in the Red Hat OpenShift Cluster Manager documentation. 
+For more information link:https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html/managing_clusters/assembly-managing-clusters#transferring-cluster-ownership_assembly-managing-clusters[about transferring cluster ownership], see "Transferring cluster ownership" in the {cluster-manager-first} documentation.
```

Comment:
Hi Eric - if it's not too late, the OCM doc link is now updated to 2022 - can that be changed still in the PR? Let me know if you need any additional details, this is the link:

https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2022/html-single/managing_clusters/index